From 9ada20309daaf3b7ca1ea721c9aea2e180f24e5d Mon Sep 17 00:00:00 2001
From: Antheas Kapenekakis <lkml@antheas.dev>
Date: Sun, 3 Aug 2025 05:16:39 +0200
Subject: [PATCH 084/104] ACPI: EC: Set ec_no_wakeup for MSI Claw A8

Same panel as the Legion Go S, and is an AMD variant.

Signed-off-by: Antheas Kapenekakis <lkml@antheas.dev>
---
 drivers/acpi/ec.c | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/drivers/acpi/ec.c b/drivers/acpi/ec.c
index 59b3d50ff..b94266ee5 100644
--- a/drivers/acpi/ec.c
+++ b/drivers/acpi/ec.c
@@ -2351,6 +2351,15 @@ static const struct dmi_system_id acpi_ec_no_wakeup[] = {
 			DMI_MATCH(DMI_PRODUCT_NAME, "83Q3"),
 		}
 	},
+	/*
+	 * MSI Claw A8. Similar issue to Legion Go S (and same screen)
+	 */
+	{
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Micro-Star International Co., Ltd."),
+			DMI_MATCH(DMI_BOARD_NAME, "MS-1T8K"),
+		}
+	},
 	{
 		// TUXEDO InfinityBook Pro AMD Gen9
 		.matches = {
-- 
2.52.0

